packages/grafana-data/src/types/data.ts

```diff
@@ -18,6 +18,7 @@ export enum LoadingState {
   Streaming = 'Streaming',
   Done = 'Done',
   Error = 'Error',
+  Warning = 'Warning',
```

Comment:
For supporting future 'Warnings' state coming from the data.

Reply:
Why? we don't have such a loading state